### PR TITLE
Make Skia port support line caps and make Cairo and Skia support miter limits

### LINKED ISSUE #141: Cairo and Skia port ignore miter values
GDIPlus port and CG port do take miter values into account however they're entirely ignored by the Skia and Cairo ports.

### PR DESCRIPTION
**Pull request by moazin**
Fixes #141 and #135. Please review @dirkschulze.

### CODE REVIEW

File: svgnative/ports/skia/SkiaSVGRenderer.cpp [hunk 1]
```diff
@@ -286,8 +313,9 @@ void SkiaSVGRenderer::DrawImage(
     SVG_ASSERT(mCanvas);
     Save(graphicStyle);
     mCanvas->clipRect({clipArea.x, clipArea.y, clipArea.x + clipArea.width, clipArea.y + clipArea.height}, SkClipOp::kIntersect);
+    SkSamplingOptions sso;
```
Comment (dirkschulze) [inline svgnative/ports/skia/SkiaSVGRenderer.cpp:316]: s/sso/samplingOptions/

**Review by dirkschulze — COMMENTED** [review]
This PR seems to have all changes from https://github.com/adobe/svg-native-viewer/pull/136 as well.

miter-limit specific changes look good to me.

**Review by dirkschulze — APPROVED** [review]
Changes look good to me. Thanks a lot! Can get merged into main.

**Comment by moazin** [comment]
@dirkschulze Thanks for reviewing. You can merge it. 👍

**Pull request merged**